bindings/core/src/method_handler/mod.rs

```diff
@@ -8,8 +8,8 @@ mod secret_manager;
 mod utils;
 mod wallet;
 
-pub use call_method::{
-    call_client_method, call_secret_manager_method, call_utils_method, call_wallet_method, CallMethod,
-};
+#[cfg(not(target_family = "wasm"))]
+pub use call_method::CallMethod;
+pub use call_method::{call_client_method, call_secret_manager_method, call_utils_method, call_wallet_method};
```

Comment:
Why is the trait non-wasm only?

Reply:
Didn't compile for wasm

Reply:
```
error: future cannot be sent between threads safely
   --> bindings/core/src/method_handler/call_method.rs:34:9
    |
34  |         Box::pin(call_client_method(self, method))
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ future returned by `call_client_method` is not `Send`
    |
    = help: within `impl futures::Future<Output = response::Response>`, the trait `std::marker::Send` is not implemented for `*mut u8`
note: future is not `Send` as this value is used across an await
   --> /home/t/Desktop/iota-sdk/sdk/src/client/api/block_builder/mod.rs:376:89
    |
376 |                 gloo_timers::future::TimeoutFuture::new((time * 50).try_into().unwrap()).await;
    |                 ------------------------------------------------------------------------^^^^^^- `gloo_timers::future::TimeoutFuture::new((time * 50).try_into().unwrap())` is later dropped here
    |                 |                                                                       |
    |                 |                                                                       await occurs here, with `gloo_timers::future::TimeoutFuture::new((time * 50).try_into().unwrap())` maybe used later
    |                 has type `gloo_timers::future::TimeoutFuture` which is not `Send`
    = note: required for the cast from `impl futures::Future<Output = response::Response>` to the object type `dyn futures::Future<Output = response::Response> + std::marker::Send`

```

Reply:
We can probably just add that Send bound to the trait

Reply:
This is what I did on some other wasm related stuff